packages/components/app/styles/components/form/sensitive-input.scss

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// SENSITIVE-INPUT
+//
+
+$hds-form-sensitive-input-button-size: 24px;
+
+.hds-form-sensitive-input {
+  position: relative;
+  width: 100%;
+
+  .hds-form-sensitive-input__control {
+    padding-right: calc(var(--token-form-control-padding) + $hds-form-sensitive-input-button-size);
+  }
+}
+
+.hds-form-sensitive-input--obfuscated {
+  .hds-form-sensitive-input__control {
+    -webkit-text-security: disc;
+  }
+}
+
+.hds-form-sensitive-input--not-obfuscated {
+  .hds-form-sensitive-input__control {
+    -webkit-text-security: none;
+  }
+}
+
+.hds-form-sensitive-input__toggle-button {
+  position: absolute;
+  top: calc(var(--token-form-control-padding) - 1px);
+  right: var(--token-form-control-padding);
```

Comment:
why the difference between these two? 🤔

Reply:
Subtracting `1px` for the TextInput/Textarea border (as the icon is positioned relative to the wrapping div, not the input). It should've been applied to the right as well. Updated in [7d5ebb9](https://github.com/hashicorp/design-system/pull/1468/commits/7d5ebb916aabcc7b69dd6d77a71061e472677b7e) and used tokens to make the computation more clear.